lib/sparkql/token.rb

```diff
@@ -10,7 +10,7 @@ module Sparkql::Token
   DECIMAL = /^\-?[0-9]+\.[0-9]+/
   CHARACTER = /^'([^'\\]*(\\.[^'\\]*)*)'/
   DATE = /^[0-9]{4}\-[0-9]{2}\-[0-9]{2}/
-  DATETIME = /^[0-9]{4}\-[0-9]{2}\-[0-9]{2}T[0-9]{2}\:[0-9]{2}\:[0-9]{2}(\.[0-9]{6})?((\+|-)[0-9]{4})?/
```

Comment:
One thing that we should at least check generally, is if the old suffix format was ever used. I think it generally is convertible to the new one with the colon.